### PR TITLE
feature: add kline close volatility indicator

### PR DESCRIPTION
**Pull request by austin362667**
via caculating standard deviation of close price.

### CODE REVIEW

File: pkg/bbgo/session.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ type StandardIndicatorSet struct {
 	ewma  map[types.IntervalWindow]*indicator.EWMA
 	boll  map[types.IntervalWindow]*indicator.BOLL
 	stoch map[types.IntervalWindow]*indicator.STOCH
+	vol   map[types.IntervalWindow]*indicator.VOL
```
Comment (c9s) [inline pkg/bbgo/session.go:38]: call it volatility ? because vol can also be volume

File: pkg/indicator/vol.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	}
+	return inc.Values[len(inc.Values)-1]
+}
+
+func (inc *VOL) calculateAndUpdate(klines []types.KLine) {
+	if len(klines) < inc.Window {
+		return
+	}
+
+	var index = len(klines) - 1
```
Comment (c9s) [inline pkg/indicator/vol.go:39]: I would call it `end`

File: pkg/indicator/vol.go [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+	var kline = klines[index]
```
Comment (c9s) [inline pkg/indicator/vol.go:40]: and this for `last` or `lastKLine`

File: pkg/indicator/vol.go [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+	if inc.EndTime != zeroTime && kline.GetEndTime().Before(inc.EndTime) {
+		return
+	}
+
+	var recentT = klines[index-(inc.Window-1) : index+1]
+
+	vol, err := calculateVOL(recentT, inc.Window, KLineClosePriceMapper)
+	if err != nil {
+		log.WithError(err).Error("VOL error")
```
Comment (c9s) [inline pkg/indicator/vol.go:50]: error: `can not calculate volatility`?

File: pkg/bbgo/session.go (hunk 1 above)
Comment (austin362667) [reply pkg/bbgo/session.go]: Sure done!

File: pkg/indicator/vol.go (hunk 2 above)
Comment (austin362667) [reply pkg/indicator/vol.go]: fixed

File: pkg/indicator/vol.go (hunk 3 above)
Comment (austin362667) [reply pkg/indicator/vol.go]: fixed

File: pkg/indicator/vol.go (hunk 4 above)
Comment (austin362667) [reply pkg/indicator/vol.go]: ok done

**Comment by c9s** [comment]
need rebase

**Pull request closed without merging**